examples/research_projects/onnxruntime/unconditional_image_generation/train_unconditional.py

```diff
@@ -552,16 +558,20 @@ def transform_images(examples):
                     generator=generator,
                     batch_size=args.eval_batch_size,
                     output_type="numpy",
-                    num_inference_steps=args.ddpm_num_steps,
+                    num_inference_steps=args.ddpm_num_inference_steps,
                 ).images
 
                 # denormalize the images and save to tensorboard
                 images_processed = (images * 255).round().astype("uint8")
 
-                if args.logger == "tensorboard":
+                if args.logger == "tensorboard" and is_tensorboard_available():
```

Comment:
let's move `is_tensorboard_available()` check to the beginning of the script next to where we do the `is_wandb_available()` check :)